### ISSUE TITLE
Language selection screen [Blocked: #20, #44]

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#52]
There needs to be a page where the user can view all languages that the Oppia app supports and select which on it should be displayed in. This should be a per-profile setting that sets the language for the app, but not for the whole system. It should be easy to navigate to this screen. It should also be possible for the user to change the language only for the profile selection page temporarily as they log in. See the PRD for specifics.

**Comment by BenHenning** [comment]
This is blocked by #20 and #44 (for where the language selection screen will be displayed).

**Comment by anandwana001** [comment]
Now we do have a screen where we can see all the languages. 
Things left to do:
1. Check and fix if the app is updating according to the selected language or not

**Comment by BenHenning** [comment]
This is a duplicate of https://github.com/oppia/oppia-android/issues/4606 (which is now tracking the request for functionality from the product team).

**Comment by BenHenning** [comment]
@KevinGitonga I think you'll need to make sure that those TODOs are addressed as part of your project.

**Comment by BenHenning** [comment]
Now that #4762 is merged, this should be fixed.

**Comment by KolliAnitha** [comment]
App language screen is now seen and it is unique to each profile
![subject (9)](https://github.com/oppia/oppia-android/assets/101634267/99bca2ea-9388-4c9d-a431-1cb5c8ccc86f)
